.github/pull_request_template.md

```diff
@@ -8,19 +8,36 @@ Please click the `Preview` tab and select a different PR template if your change
 ## Describe your changes (can list changeset entries if preferable)
 
 
-## Author Checklist (complete before requesting a review)
-- [ ] I have performed a self-review of my code
-- [ ] I have added thorough tests where applicable (unit / component / visual)
-- [ ] I have reviewed the `PIE Storybook`/`PIE Docs` PR preview
-- [ ] I have reviewed visual test updates properly before approving
+## Author Checklist (complete before requesting a review, do not delete any)
+- [ ] I have performed a self-review of my code.
+- [ ] I have added thorough tests where applicable (unit / component / visual).
+- [ ] I have reviewed the `PIE Storybook`/`PIE Docs` PR preview.
+- [ ] I have reviewed visual test updates properly before approving.
 - [ ] If changes will affect consumers of the package, I have created a changeset entry.
-- [ ] If a changeset file has been created, I have used the `/snapit` functionality to test my changes in a consuming application
+- [ ] If a changeset file has been created, I have tested these changes in [PIE Aperture](https://github.com/justeattakeaway/pie-aperture/) using the `/test-aperture` command.
+
+## Not-applicable Checklist items
```

Comment:
I'm guessing we'll need to add something to danger to not check this section so it doesn't moan at us?

Reply:
yes! trying an update now - good spot